src/server/game/Spells/Auras/SpellAuraEffects.cpp

```diff
@@ -3270,17 +3270,11 @@ void AuraEffect::HandleAuraMounted(AuraApplication const* aurApp, uint8 mode, bo
             }
         }
 
-        if (CreatureTemplate const* creatureInfo = sObjectMgr->GetCreatureTemplate(creatureEntry))
+        if (CreatureTemplate const* creatureInfo = sObjectMgr->GetCreatureTemplate(creatureEntry)) // FIGURE OUT HOW TO FIX THIS PART
```

Comment:
?

Reply:
Not sure what the issue is/was there or if I figured it out. I'll delete the comment and we should make a note to test mount aura spells and ensure they...work.

Reply:
shouldnt be hard to test at least